<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgManageReleases</class>
 <widget class="QDialog" name="dlgManageReleases">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="vtitestmanager.qrc">
    <normaloff>:/images/releases.png</normaloff>:/images/releases.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QGroupBox" name="gbReleases">
       <property name="title">
        <string> Release location </string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lblCurrentDir">
          <property name="text">
           <string>Current directory :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledCurrentDir">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbAIFChangeDir">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Change directory</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="vtitestmanager.qrc">
            <normaloff>:/images/directory.png</normaloff>:/images/directory.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbInformation">
       <property name="title">
        <string> System information </string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="lblRegAIFVersion">
          <property name="text">
           <string>Current version :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="ledRegAIFVersion">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblRegInnovaPrg">
          <property name="text">
           <string>Innova Program :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="ledRegInnovaPrg">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="lblITUVersion">
          <property name="text">
           <string>ITU Version :</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLineEdit" name="ledITUVersion">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lblFootSwitchType">
          <property name="text">
           <string>Footswitch type :</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="ledFootSwitchType">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="lblRegIPAW">
          <property name="text">
           <string>AW PC IP Address :</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLineEdit" name="ledRegIPAW">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="lblRegIPEng">
          <property name="text">
           <string>ENG PC IP Address :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="ledRegIPEng">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="ledRegFPProgram">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="lblRegFPProgram">
          <property name="text">
           <string>FluoroPlus Program :</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbAIF">
       <property name="title">
        <string> AIF </string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="lblAIFAvailableReleases">
              <property name="text">
               <string>Available releases :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listAIFReleases">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>64</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>64</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>12</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pbAIFInstall">
              <property name="text">
               <string>Install selected release</string>
              </property>
              <property name="icon">
               <iconset resource="vtitestmanager.qrc">
                <normaloff>:/images/releases.png</normaloff>:/images/releases.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbAIFUninstall">
              <property name="text">
               <string>Uninstall current release</string>
              </property>
              <property name="icon">
               <iconset resource="vtitestmanager.qrc">
                <normaloff>:/images/releases.png</normaloff>:/images/releases.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>5</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbCycler">
       <property name="title">
        <string> Cycler </string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="lblCyclerAvailableReleases">
              <property name="text">
               <string>Available releases :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listCyclerReleases">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>64</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>64</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <spacer name="verticalSpacer_5">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>12</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pbCyclerInstall">
              <property name="text">
               <string>Install selected release</string>
              </property>
              <property name="icon">
               <iconset resource="vtitestmanager.qrc">
                <normaloff>:/images/releases.png</normaloff>:/images/releases.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbCSS">
       <property name="title">
        <string> 5 Min. Check </string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="lblCSSAvailableReleases">
              <property name="text">
               <string>Available releases :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listCSSReleases">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>64</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>64</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <spacer name="verticalSpacer_6">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>12</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pbCSSInstall">
              <property name="text">
               <string>Install selected release</string>
              </property>
              <property name="icon">
               <iconset resource="vtitestmanager.qrc">
                <normaloff>:/images/releases.png</normaloff>:/images/releases.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_7">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <widget class="QPushButton" name="pbExit">
       <property name="text">
        <string>Exit</string>
       </property>
       <property name="icon">
        <iconset resource="vtitestmanager.qrc">
         <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>174</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbInstallAll">
       <property name="text">
        <string>Install selected releases</string>
       </property>
       <property name="icon">
        <iconset resource="vtitestmanager.qrc">
         <normaloff>:/images/releases.png</normaloff>:/images/releases.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_8">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ledRegAIFVersion</tabstop>
  <tabstop>ledRegInnovaPrg</tabstop>
  <tabstop>listAIFReleases</tabstop>
  <tabstop>pbAIFInstall</tabstop>
  <tabstop>pbAIFUninstall</tabstop>
  <tabstop>pbExit</tabstop>
 </tabstops>
 <resources>
  <include location="vtitestmanager.qrc"/>
 </resources>
 <connections/>
</ui>
